config/owasp/suppressions.xml

```diff
@@ -8,7 +8,7 @@
     <cpe>cpe:/a:pivotal_software:spring_security</cpe>
     <cve>CVE-2018-1258</cve>
   </suppress>
-  <suppress until="2021-11-30">
+  <suppress until="2022-01-05">
```

Comment:
Seems reasonable to extend it beyond the usual one month given the Christmas/New Year break

Reply:
More info: 
https://github.com/jeremylong/DependencyCheck/issues/3623

Reply:
Alex, I looked at there before, but could not see any solution. Do you have a suggestion, objection to merge this?